modules/dxp/apps/portal-search-tuning/portal-search-tuning-rankings-web/src/main/resources/META-INF/resources/js/ResultRankingsApp.es.js

```diff
@@ -13,12 +13,15 @@ import React from 'react';
 
 import ThemeContext from './ThemeContext.es';
 import ResultRankingsForm from './components/ResultRankingsForm.es';
+import ErrorBoundary from './components/shared/ErrorBoundary.es';
 
 export default function({context, props}) {
 	return (
 		<ThemeContext.Provider value={context}>
 			<div className="results-rankings-root">
-				<ResultRankingsForm {...props} />
+				<ErrorBoundary component="Result Rankings">
```

Comment:
```suggestion
				<ErrorBoundary component={Liferay.Language.get('result-rankings')}>
```

Reply:
Let's also update this for the rest of the uses of `ErrorBoundary` with the `component` prop.